docker-compose.yml

```diff
@@ -1,133 +1,102 @@
-# This file is designed for production server deployment, not local development work
+# This is a Docker Compose file for deploying Mastodon. It is designed to be used with the `docker compose` command.
+# This file is designed as a starting point for your production server deployment, not local development work!
 # For a containerized local dev environment, see: https://github.com/mastodon/mastodon/blob/main/README.md#docker
 
-services:
-  db:
-    restart: always
-    image: postgres:14-alpine
-    shm_size: 256mb
-    networks:
-      - internal_network
-    healthcheck:
-      test: ['CMD', 'pg_isready', '-U', 'postgres']
-    volumes:
-      - ./postgres14:/var/lib/postgresql/data
-    environment:
-      - 'POSTGRES_HOST_AUTH_METHOD=trust'
+# You should check the Docker image tag used for the web, sidekiq, and streaming containers to make sure you are running
+# the appropriate version for your deployment. Using `nightly` will deploy the in-development version of Mastodon, which is
+# not considered a stable release. Visit https://github.com/mastodon/mastodon/releases for currently released versions and tags.
+x-image-tag:
+  &image-tag nightly
```

Comment:
Maybe add a comment here to explain this is the place to change the version for updates